docs/FAQ/how-deploy-bold-bi-in-eks-using-network-load-balancer-with-ssl-certificate-from-acm.md

```diff
@@ -0,0 +1,245 @@
+# How to deploy Bold BI in EKS Using Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
+
+The below section explains how to deploy Bold BI in EKS using Nginx Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
+
+## Create a EKS Cluster and filestorage:
+
+1. Create an Amazon EKS cluster and [node group](https://docs.aws.amazon.com/eks/latest/userguide/eks-compute.html) to deploy Bold BI.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html 
+
+2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following the below link.
+
+   ![AWS EFS](../images/aws-efs.png)
+
+3. Connect to your Amazon EKS cluster.
+   https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/
+  
+4. Deploy the EFS CSI Driver to your cluster.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html 
+
+	```console
+	kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.4"
+	```
+	
+	> <b>Note:</b> Make sure the following network configurations to avoid volume mounting issues.<br/><br/>
+		1. EFS file system in the same region as your EKS cluster.<br/>
+		2. Your EKS cluster and EFS file system must be in the same Virtual Private Cloud (VPC).<br/>
+		3. The security group for your EFS file system, which will allow incoming traffic from your EKS cluster's nodes.<br/>
+		4. Ensure the EFS CSI Driver installation in cluster.<br/>
+
+## Deploy Network Load Balancer with SSL termination:
+
+1. Download the [deploy.yaml](https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.5.1/deploy/static/provider/aws/nlb-with-tls-termination/deploy.yaml) template
+
+2. Edit the `deploy.yaml` file and change the VPC CIDR in use for the Kubernetes cluster.
```

Comment:
2. Edit the `deploy.yaml` file and change the VPC CIDR for the Kubernetes cluster.

Reply:
concern addressed